arithmetic_03/even_multiple_of_3.py

```diff
@@ -3,4 +3,11 @@
 
 
 def even_multiple_of_3(number):
-    pass
+    if number % 3 == 0:
```

Comment:
We should be checking for both divisibility and whether the number is even or not